Sources/Penny/SwiftReleasesChecker.swift

```diff
@@ -0,0 +1,85 @@
+import Logging
+import DiscordBM
+import Collections
+#if canImport(FoundationEssentials)
+import FoundationEssentials
+#else
+import Foundation
+#endif
+
+actor SwiftReleasesChecker {
+    struct Storage: Sendable, Codable {
+        var currentReleases: [SwiftOrgRelease] = []
+    }
+
+    var storage = Storage()
+
+    let swiftReleasesService: any SwiftReleasesService
+    let discordService: DiscordService
+    let logger = Logger(label: "SwiftReleasesChecker")
+
+    init(swiftReleasesService: any SwiftReleasesService, discordService: DiscordService) {
+        self.swiftReleasesService = swiftReleasesService
+        self.discordService = discordService
+    }
+
+    nonisolated func run() {
+        Task { [self] in
```

Comment:
Why are we doing this with unstructured concurrency?

Reply:
I was thinking about this. I should move things to ServiceLifecycle, but that'll have to be for another PR.

Reply:
Everything matches ServiceLifecycle too, coincidentally. Even the function name, other than the fact that the function is a "blocking" function which is what ServiceLifecycle expects.

Reply:
This pattern is what we use everywhere in Penny so can't just fix it in one place, Will need more changes.

Reply:
https://github.com/vapor/penny-bot/pull/255